discovery/gossiper_test.go

```diff
@@ -3195,7 +3194,8 @@ func TestSendChannelUpdateReliably(t *testing.T) {
 	require.NoError(t, err, "unable to process remote channel proof")
 
 	// Now that we've constructed our full proof, we can assert that the
-	// channel has been announced.
+	// channel has been announced. We anticipate the local ChanAnnouncement
+	// and the local ChanUpdate.
```

Comment:
Nice clarification here. I do wonder if instead of just counting the messages though that we should actually check that both of those distinct messages appear

Reply:
good idea will add the msg check.